### DISCUSSION TITLE
Scholarly gives "Cannot Fetch from Google Scholar"

### DISCUSSION THREAD

**Discussion by mateosere** [scholarly-python-package/scholarly#474] [Q&A] [answered]
I'm trying to get data from Google Scholar

```
from scholarly import scholarly

search_query = scholarly.search_author('Albert Einstein')
author = next(search_query).fill()

```
and the output that I obtain is:

`MaxTriesExceededException: Cannot Fetch from Google Scholar.
`
Any idea of what it's going wrong?

I'm using scholarly == 1.7.6 and I'm using a VPN so it shouldn't be an IP problem.

**Comment by arunkannawadi** [comment, marked as answer]
Please see #465
